packages/financial-templates-lib/src/clients/FinancialContractClient.js

```diff
@@ -260,7 +260,7 @@ class ExpiringMultiPartyClient {
     });
     this.lastUpdateTimestamp = currentTime;
     this.logger.debug({
-      at: "ExpiringMultiPartyClient",
+      at: "FinancialContractClient",
       message: "Expiring multi party state updated",
```

Comment:
does this message need to change?

Reply:
properly! good catch. will do a find and replace for all `Expiring multi party` and associated changes.